src/main/kotlin/org/taktik/icure/asynclogic/ContactLogic.kt

```diff
@@ -60,7 +60,8 @@ interface ContactLogic : EntityPersister<Contact, String> {
     fun listContactsByHcPartyServiceId(hcPartyId: String, formId: String): Flow<Contact>
     fun listContactsByExternalId(externalId: String): Flow<Contact>
     fun listServicesByAssociationId(associationId: String): Flow<Service>
-    fun listServicesForHealthElementId(healthElementId: String): Flow<Service>
+    fun listServicesForHealthElementIds(hcPartyId: String, healthElementIds: List<String>): Flow<Service>
+    fun listServiceIdsForHealthElementIds(hcPartyId: String, healthElementIds: List<String>): Flow<String>
```

Comment:
```suggestion
    fun listServiceIdsByHealthElementIds(hcPartyId: String, healthElementIds: List<String>): Flow<String>
```
To keep consistency with the DAO